.changeset/tricky-shrimps-guess.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Is there no publishing of the shopping list package yet? This didn't come up as a package when I created the changeset.